lang/python/python-platformio/Makefile

```diff
@@ -0,0 +1,62 @@
+# This is free software, licensed under the GNU General Public License v2.
+# See /LICENSE for more information.
+#
+
+include $(TOPDIR)/rules.mk
+
+PKG_NAME:=python-platformio
+PKG_VERSION:=6.1.16
+PKG_RELEASE:=1
+
+PYPI_NAME:=platformio
+PYPI_SOURCE_NAME:=platformio
```

Comment:
Isnt this same value as in PYPI_NAME? It looks like, so this one can be removed.
```suggestion
```
Based on https://github.com/openwrt/packages/tree/master/lang/python#building-a-python-package